packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -183,12 +191,38 @@ enum OutputView {
 	Data = 'Data'
 }
 
+const showSaveDataDialog = ref<boolean>(false);
 const view = ref(OutputView.Charts);
 const viewOptions = ref([
 	{ value: OutputView.Charts, icon: 'pi pi-image' },
 	{ value: OutputView.Data, icon: 'pi pi-list' }
 ]);
 
+const hiddenDialog = () => {
+	showSaveDataDialog.value = false;
+};
```

Comment:
Prefer more consistency, down below in menuItems you are manipulating `showSaveDataDialog` variable directly without passing through a proxy-function.

Slight preference just to do direct manipulation since the idea of the variable is quite simple, if you want to use functions prefer usage of verbs, e.g. "hideDialog" instead of "hiddenDialog".